content/en/legacy/flux/references/garbagecollection.md

```diff
@@ -8,15 +8,15 @@ Part of syncing a cluster with a git repository is getting rid of
 resources in the cluster that have been removed in the repository. You
 can tell `fluxd` to do this "garbage collection" using the command-line
 flag `--sync-garbage-collection`. It's important to know how it
-operates, and appreciate its limitations, before enabling it.
+operates and appreciate its limitations before enabling it.
 
 ## How garbage collection works
 
 When garbage collection is enabled, syncing is done in two phases:
 
  1. Apply all the manifests in the git repo (as delimited by the
-    branch and path arguments), and give each resource a label marking
```

Comment:
AIUI this is the main point of the PR, I assume you looked for this label and spent longer than you needed to trying to find it; and well, if we're going to mention it, why not name it outright.

Then otherwise the grammar and spelling corrections, but mainly this (?)

Reply:
Flux v1 is in maintenance mode, but the repo is not archived, and docs still live in website, so I don't see any reason why we can't merge these fixes.

At some point we will stop accepting changes to Helm Operator and Flux v1 docs, (but we are not there yet)

Reply:
Yup, that's exactly what happened